assets/evaluation_on_cloud/environments/evaluations-built-in/asset.yaml

```diff
@@ -0,0 +1,6 @@
+name: built-in-evaluator-env
```

Comment:
```suggestion
name: built-in-evaluator
```
I believe you want to omit "env"?

Reply:
Good catch, I modified it to "evaluator-built-in" like the other naming pattern. Thanks @lykelly19